### PR TITLE
fix(euler): fix token values and calculations

### PR DESCRIPTION
**Pull request by conwayconstar**
## Description

Various fixes for euler app integration

- Fixed TVL
- Fixed E, D, P token prices
- Removed Supply APY from D tokens
- Removed Borrow APY from E tokens
- Removed Borrow/Supply APY from P tokens
- Updated logo

## Checklist

- [x] I have followed the [Contributing Guidelines](../CONTRIBUTING.md)
- [ ] (optional) As a contributor, my Ethereum address/ENS is:
- [ ] (optional) As a contributor, my Twitter handle is:

### CODE REVIEW

File: src/apps/euler/ethereum/euler.e-token.token-fetcher.ts [hunk 1]
```diff
@@ -85,47 +84,42 @@ export class EthereumEulerETokenTokenFetcher implements PositionFetcher<AppToken
...
                 value: buildDollarDisplayItem(dataProps.liquidity),
               },
-              {
-                label: 'Borrow APY',
-                value: buildDollarDisplayItem(dataProps.borrowAPY),
-              },
               {
                 label: 'Supply APY',
-                value: buildDollarDisplayItem(dataProps.supplyAPY),
+                value: dataProps.supplyAPY,
```
Comment (immasandwich) [inline src/apps/euler/ethereum/euler.e-token.token-fetcher.ts:122]: Use `buildPercentageDisplayItem`

File: src/apps/euler/ethereum/euler.e-token.token-fetcher.ts [hunk 2]
```diff
@@ -85,47 +84,42 @@ export class EthereumEulerETokenTokenFetcher implements PositionFetcher<AppToken
...
         return {
           address: market.eTokenAddress,
           symbol: `E${market.symbol}`,
-          name: `Euler E token ${market.name}`,
+          name: `${market.name} (E)`,
           type: ContractType.APP_TOKEN as const,
-          supply: Number(market.totalSupply) / 10 ** Number(market.decimals),
-          pricePerShare,
+          supply: Number(market.totalBalances) / 10 ** Number(market.decimals),
+          pricePerShare: underlyingToken.price * Number(utils.formatEther(pricePerShare)),
```
Comment (immasandwich) [inline src/apps/euler/ethereum/euler.e-token.token-fetcher.ts:105]: Should be `1` (pricePerShare is the ratio between the token and the underlying token price)

See https://studio.zapper.fi/docs/concepts/app-tokens#what-is-pricepershare-why-is-it-useful

File: src/apps/euler/ethereum/euler.d-token.token-fetcher.ts [hunk 3]
```diff
@@ -84,27 +84,26 @@ export class EthereumEulerDTokenTokenFetcher implements PositionFetcher<AppToken
...
         return {
           address: market.dTokenAddress,
           symbol: `D${market.symbol}`,
-          name: `Euler D token ${market.name}`,
+          name: `${market.name} (D)`,
           type: ContractType.APP_TOKEN as const,
-          supply: Number(market.totalSupply) / 10 ** Number(market.decimals),
-          pricePerShare: 1,
+          supply: Number(market.totalBalances) / 10 ** Number(market.decimals),
+          pricePerShare: underlyingToken.price,
```
Comment (immasandwich) [inline src/apps/euler/ethereum/euler.d-token.token-fetcher.ts:99]: Should be 1, see the comment below.

**Review by immasandwich — APPROVED** [review]

**Comment by conwayconstar** [comment]
> @conwayconstar I can't merge because there's a few merge conflicts.. can you merge the latest from `main`? I'll get this in as soon as you can do that.

I have merged in main

**Comment by immasandwich** [comment]
You'll likely find this document helpful:
https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork

What I would do is simply:

```
git remote add upstream https://github.com/Zapper-fi/studio.git
git fetch upstream
git merge upstream/main
```

This will fetch the latest branches from the upstream repo, then merge the `upstream/main` branch into your branch. Then, you can resolve conflicts and push. You can also `git checkout master && git merge upstream/main` to get your forked `main` branch in sync.

**Pull request merged**